changelog.d/8802.feature

```diff
@@ -0,0 +1 @@
+The prometheus rules are updated to include missing rules the grafana dashboard uses.
```

Comment:
```suggestion
Fix the "Event persist rate" section of the included grafana dashboard by adding missing prometheus rules.
```

We tend to aim our newsfragments towards sysadmins, and this might give them a better idea of what exactly this change fixes. Mostly just a super short summary of what you already wrote in the PR description :)

Reply:
I feel this should be doc instead. This kinda counts as documentation?

Reply:
Yes, i was very unsure about the changelog wording and category, i updated it with the suggestion and moved it to `.doc` now.